cmd/juju/cloud/detectcredentials.go

```diff
@@ -101,6 +101,7 @@ func (c *detectCredentialsCommand) Info() *cmd.Info {
 		Name:    "autoload-credentials",
 		Purpose: detectCredentialsSummary,
 		Args:    "[<cloud-type>]",
+		Aliases: []string{"autoload-credential"},
```

Comment:
This alias is wrong - it really is autoload **all** the credentials found on the machine (not just one) so plural is absolutely correct here. juju autoload-credential makes no sense

Reply:
Reverted as discussed.